ssv/spectest/generate/state_comparison/*synccommitteeaggregator.SyncCommitteeAggregatorProofSpecTest/sync committee aggregator all are aggregators.json

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
I think my PR shouldn't be adding new state comparison jsons but just change existing ones (so we see what's changed)

Reply:
The test was broken
When https://github.com/bloxapp/ssv-spec/pull/281 gets merged (soon I hope), you will see a diff.